tests/test_build.py

```diff
@@ -89,3 +93,40 @@ def test_android_manifest_xml(self):
         assert xml.count('android:debuggable="true"') == 1
         assert xml.count('<service android:name="abcd" />') == 1
         # TODO: potentially some other checks to be added here to cover other "logic" (flags and loops) in the template
+
+
+class TestContext(unittest.TestCase):
+
+    @mock.patch.dict('pythonforandroid.build.Context.env')
+    @mock.patch('pythonforandroid.build.get_available_apis')
+    @mock.patch('pythonforandroid.build.ensure_dir')
+    def test_sdk_ndk_paths(
+            self,
+            mock_ensure_dir,
+            mock_get_available_apis,
+    ):
+        mock_get_available_apis.return_value = [RECOMMENDED_TARGET_API]
+        context = Context()
+        context.setup_dirs(os.getcwd())
+        context.prepare_build_environment(
+            user_sdk_dir='sdk',
+            user_ndk_dir='ndk',
+            user_android_api=None,
+            user_ndk_api=None,
+        )
+
+        real_sdk_dir = os.path.join(os.getcwd(), 'sdk')
+        real_ndk_dir = os.path.join(os.getcwd(), 'ndk')
+        assert context.sdk_dir == real_sdk_dir
+        assert context.ndk_dir == real_ndk_dir
```

Comment:
Super-minor, can we add a comment describing why is that needed?
Is very likely that in the future we will not immediately get why that was needed.

Reply:
Sure.